lib/src/stdio.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'stdio_group.dart';
+
+/// An opaque key for the Zone value that contains the [StdioGroup] for the
+/// current captured stdout context.
+final stdoutKey = Object();
```

Comment:
Would it make sense to have some sort of meaningful objects here for debugging purposes? Or is there some restriction on what can be used as keys for zone values?